src/configuration.h

```diff
@@ -134,7 +134,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 // The SH1106 controller is almost, but not quite, the same as SSD1306
 // Define this if you know you have that controller or your "SSD1306" misbehaves.
-//#define USE_SH1106
+#define USE_SH1106
 
```

Comment:
Please revert that change, the type of controller (1106/1306) is auto-detected now if it is present on the i2c bus. see https://github.com/meshtastic/Meshtastic-device/blob/a8ef3967cde22993608c7adba051b3cea502ba30/src/debug/i2cScan.h#L6-L25